File: achilles/03-19-2024/modelA.py
def print_details(name, age, **other_info):
    print("Name:", name)
    print("Age:", age)

    for key, value in other_info.items():
        print(key, ":", value)

# Calling the function with keyword arguments:
print_details("Alice", 30, city="New York", occupation="Software Engineer")

def calculate(num1, num2, operation="add"):  # operation has a default value
    if operation == "add":
        return num1 + num2
    elif operation == "multiply":
        return num1 * num2
    else:
        return "Invalid operation"

# Using positional and keyword arguments
result = calculate(10, 5, operation="multiply")  
print(result)  # Output: 50

File: codex/04-16-2024/prompt-01/modelA.java
public class modelA {
  public static <T> void printElement(T element) {
    System.out.println("Element: " + element);
  }

  public static void main(String[] args) {
    String name = "Alice";
    int age = 30;

    printElement(name); // Output: Element: Alice
    printElement(age);   // Output: Element: 30
  }
}